src/Xabe.FFmpeg/Conversion/Conversion.cs

```diff
@@ -281,29 +284,79 @@ public IConversion SetOutput(string outputPath)
             OutputFilePath = outputPath;
             _output = $"\"{outputPath}\"";
             return this;
+        }
+        /// <inheritdoc />
+        public IConversion SetVSync(int vsyncmode)
+        {
+            if (vsyncmode != -1) AddParameter($"-vsync {vsyncmode}");

```

Comment:
Change add parameter to assign value to variable (e.g. "vsyncMode").
If user "Set" vsync two times he will get an error.